src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/media-edit-overlay/main.js

```diff
@@ -344,6 +418,35 @@ define([
         },
 
         /**
+         * Starts the dropzone for changing the preview image
+         */
+        startPreviewDropzone: function() {
+            this.sandbox.on('husky.dropzone.preview-img.files-added', this.newVersionUploadedHandler.bind(this));
+
+            this.sandbox.start([
+                {
+                    name: 'dropzone@husky',
+                    options: {
+                        el: constants.previewDropzoneSelector,
+                        maxFilesize: config.get('sulu-media').maxFilesize,
+                        url: '/admin/api/media/preview/' + this.media.id,
+                        method: 'POST',
+                        paramName: 'previewImg',
+                        showOverlay: false,
+                        skin: 'overlay',
+                        titleKey: '',
+                        descriptionKey: 'sulu.media.upload-new-preview',
+                        instanceName: 'preview-img',
```

Comment:
We try to avoid abbreviations like `img` (same for the `paramName` a few lines earlier)